app/feeswapper/management/commands/feeSwapper.py

```diff
@@ -0,0 +1,185 @@
+                # Check if API call returned any ERC-20 tokens associated with the FEE_ADDRESS.  Return null token list if not
+                if r.json()['status'] != '1':
+                        return {}
+                for transaction in r.json()['result']:
+                        address = transaction['contractAddress']                      
+                        exchangeAddress = self.factoryContract.functions.getExchange(self.web3.toChecksumAddress(address)).call()
+                        if exchangeAddress != '0x0000000000000000000000000000000000000000':
+                                # Only add token to list to ETH if Uniswap exchange exists
+                                tokenList[address]={'tokenName':transaction['name'],'tokenSymbol':transaction['symbol'],'exchangeAddress':exchangeAddress,'balance':transaction['balance']}
+                                print('Token Name: ' + tokenList[address]['tokenName']+ ' and Token Symbol is: ' + tokenList[address]['tokenSymbol'])
+                                print('Token address is: ' + address)
+                                print('Uniswap Exchange address is: '+ exchangeAddress)
+                                print('Token balance is: ' + tokenList[address]['balance'])
+                return tokenList
+
+        def sell_token(self, exchangeAddress, tokenSymbol):
+                """Swap total balance of ERC-20 token associated with Uniswap exchange address to ETH
+                
+                Args:
+                        exchangeAddress (str): The address of the Uniswap exchange contract assocciated with the ERC-20 token to convert to ETH
+                        tokenSymbol (str): The symbol for the ERC-20 token to be converted to ETH. 
+                
+                Note: This currently only works with the BAT Uniswap exchange on Rinkeby Testnet
+                """
+                if (self.network == 'rinkeby'):
+                        chain = 4
+                else:
+                        chain = 1
+                # Follows Uniswap doc guidance on calculating conversion
+                exchangeContract = self.web3.eth.contract(address = exchangeAddress, abi = exchangeABI)
+                tokenAddress = exchangeContract.functions.tokenAddress().call()
+                tokenContract = self.web3.eth.contract(address = tokenAddress, abi = tokenABI)
+                walletBalance = tokenContract.functions.balanceOf(settings.FEE_ADDRESS).call()
+                if walletBalance == 0:
+                        return
+                outputReserve = self.web3.eth.getBalance(exchangeAddress)
+                inputReserve = tokenContract.functions.balanceOf(exchangeAddress).call()
+                numerator = walletBalance * outputReserve * 997
+                denominator = inputReserve * 1000 + walletBalance * 997
+                outputAmount = numerator / denominator
+                deadline = self.web3.eth.getBlock('latest')['timestamp']+300
```

Comment:
Add spaces for mathematical operations

Reply:
Done locally. Will include in next commit.